modules/buckets/README.md

```diff
@@ -0,0 +1,111 @@
+3. [Monitoring](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-monitoring-cos)
+4. Data retention, [lifecycle](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-archive) and archiving options
+
+This module extends the root modules ability to create a single bucket, allowing multiple buckets to be defined and created with a single module call. All options available on the root module are supported. Examples are included in the basic, existing_resources and fscloud examples. The advantage of the root module, is the simplicity of creating an instance and a single bucket, it offer ease of use for small deployments. The advantage of the buckets module is the ability to create all required buckets in a single place. With scale comes complexity and the ability to keep the entire definition in one place helps retain clarity on the bucket properties. The outputs for all the buckets are included in a single map, again improving clarity and enabling the results to be iterated over.
+
+## Usage
+```hcl
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX" # pragma: allowlist secret
+  region           = "us-south"
+}
+
+# Create:
+# - COS buckets, one with encryption and another with versioning
+module "buckets" {
+  # Replace "main" with a GIT release version to lock into a specific release
+  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cos.git//modules/buckets?ref=main"
+  resource_group_id = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
+  bucket_configs = [
+    {
+      bucket_name            = "my-encrypted-bucket"
+      kms_key_crn            = "crn:v1:bluemix:public:kms:us-south:a/xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX:xxxxxx-XXXX-XXXX-XXXX-xxxxxx:key:xxxxxx-XXXX-XXXX-XXXX-xxxxxx"
+      kms_encryption_enabled = true # Must be set, as well as passing key crn, to avoid terraform count issues during plan and apply
+      region_location        = "us-south"
+      resource_instance_id   = "crn:v1:bluemix:public:cloud-object-storage:global:a/xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX:xxxxxxxx-XXXX-XXXX-XXXX-xxxxxxxx::"
+    },
+    {
+      bucket_name            = "my-versioned-bucket"
+      kms_encryption_enabled = false
+      region_location        = "us-south"
+      resource_instance_id   = ""crn:v1:bluemix:public:cloud-object-storage:global:a/xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX:xxxxxxxx-XXXX-XXXX-XXXX-xxxxxxxx::""
+      object_versioning = {
+        enable = true
+      }
+    }
+  ]
+}
+```
+
+## Known issues
+
+## Required IAM access policies
+
+You need the following permissions to run this module.
+
+- Account Management
+    - **Resource Group** service
+        - `Viewer` platform access
+- IAM Services
+    - **IBM Cloud Object Storage** service
+        - `Editor` platform access
+        - `Manager` service access
+
+
+<!-- BEGIN EXAMPLES HOOK -->
+## Examples
```

Comment:
These examples aren't in the submodule, so don't belong here, right?

Reply:
I would remove the examples section (and the hooks metadata so they don't get re-added) from here

Reply:
Looks like the examples other docs hooks are still in the readme.

Reply:
Examples section is still here. Is that correct?

Reply:
Examples should be removed from here I think